### ISSUE TITLE
Broken Compat with Pydantic latest

### ISSUE THREAD

**Issue by miili** [Maillol/aiohttp-pydantic#33]
The `Group` parameters are broken with the latest pydantic version.

```python
from __future__ import annotations

import logging
from aiohttp import web
from aiohttp_pydantic import PydanticView
from aiohttp_pydantic.injectors import Group

logging.basicConfig(level=logging.DEBUG)


class Test(Group):
    name: str
    age: int = 123


class TestView(PydanticView):
    async def get(self, test: Test):
        return web.Response(text=test.name)


app = web.Application()
app.router.add_view("/test", TestView)
web.run_app(app)

```

Query to `http://0.0.0.0:8080/test?name=foo` returns

```json
[{"loc": ["test"], "msg": "field required", "type": "value_error.missing", "in": "query string"}]
```

**Comment by Maillol** [comment]
The issue is not the latest pydantic version but the usage of `from __future__ import annotations`

**Comment by Maillol** [comment]
Fixed in release [1.12.1](https://pypi.org/project/aiohttp-pydantic/1.12.1/)
Thanks
